chef-utils/lib/chef-utils/dist/org.rb

```diff
@@ -0,0 +1,15 @@
+module ChefUtils
```

Comment:
Can we keep this class structure, but stick it all into a singel file so it's easier to read and require?

Reply:
arg, I had it like that at one point too XD I'll fix it, should be much easier now that I've done an initial pass.